NewHorizons/Builder/Props/TornadoBuilder.cs

```diff
@@ -142,14 +143,29 @@ private static void MakeTornado(GameObject planetGO, Sector sector, PropModule.T
 
             tornadoGO.GetComponentInChildren<CapsuleShape>().enabled = true;
 
+            var fluidType = FluidVolume.Type.CLOUD;
+
+            try
+            {
+                fluidType = (FluidVolume.Type)Enum.Parse(typeof(FluidVolume.Type), Enum.GetName(typeof(FluidType), info.fluidType).ToUpper());
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError($"Couldn't parse fluid volume type [{info.fluidType}]:\n{ex}");
+            }
+
             // Resize it so the force volume goes all the way up
             switch (downwards)
             {
                 case true:
-                    tornadoGO.transform.Find("MockDownTornado_FluidCenter").localScale = new Vector3(1, 2f, 1);
```

Comment:
We could simplify this entire switch statement to just be something like `var FluidGO = tornadoGO.transform.Find(downwards ? "MockDownTornado_FluidCenter" : "MockUpTornado_FluidCenter")` and then apply the scale and fluid volume changes to that.